package.json

```diff
@@ -4,8 +4,10 @@
   "description": "A reliable way to get env",
   "main": "lib/index.js",
   "scripts": {
-    "test": "apm test",
-    "compile": "ucompiler go"
+    "test": "(apm test) && (flow check) && (eslint . )",
```

Comment:
Maybe move the lint tasks to a different task, and then call that here if you still want that in `test`?